docs/deployment.md

```diff
@@ -33,10 +33,10 @@ Group Project XBlock v2 is done the same way other XBlocks are installed:
 2. Switch to the `edxapp` virtualenv
 3. pip install -e git+https://github.com/open-craft/xblock-group-project-v2.git@ **version_hash** #egg=xblock-group-project-v2
 
-*version_hash* is the hash of the git commit. At the time of writing, the latest stable version was
-`4322ca8092c5385d5602143077e388017fcb1249`, so the above command looked like this:
+*version_hash* is the hash of a git commit, or the name of a git tag. At the time of writing, the latest stable version was
+`0.4.7`, so the above command looked like this:
```

Comment:
There was [a version bump yesterday](https://github.com/open-craft/xblock-group-project-v2/commit/921975c2cad89ef9865acc37b7356ac8a73b5f7d) so this bit (and the one below) could be updated to point to version 0.4.9 :)